public/css/user/home.css

```diff
@@ -3,6 +3,266 @@
   padding: 0;
 }
 
+html {
+  scroll-behavior: smooth;
+}
+
+body {
+  overflow-x: hidden;
+}
+
+section {
+  height: 100vh;
+}
+
+.main--header {
+  display: flex;
+  flex-direction: column;
+  position: relative;
+}
+
+.main--header-title {
+  margin-top: 150px;
+  width: 100%;
+  display: flex;
+}
+
+.main--header-titleWhite {
+  font-weight: 300;
+  font-family: Roboto;
+  display: flex;
+  background: rgb(0, 131, 38);
+  color: white;
+  width: 500px;
+  height: 200px;
+  justify-content: center;
+  align-items: center;
+  font-size: 72px;
+}
+
+.main--header-titleGreen {
+  font-weight: 300;
+  font-family: Roboto;
+  display: flex;
+  background: white;
+  color: #008326;
+  width: 550px;
+  height: 200px;
+  justify-content: center;
+  align-items: center;
+  font-size: 72px;
+}
+
+.main--header-star {
+  position: absolute;
+  left: 80%;
+  top: 10%;
+  width: 27%;
+  height: 52%;
+}
+
+.main--header-starGreen {
+  position: absolute;
+  clip-path: polygon( 50% 0%, 61% 35%, 98% 35%, 68% 57%, 79% 91%, 50% 70%, 21% 91%, 32% 57%, 2% 35%, 39% 35%);
+  width: 500px;
+  height: 500px;
+  background: rgba(0, 131, 38, 0.9);
+  z-index: 1;
+}
+
+.main--header-starGray {
+  position: absolute;
+  clip-path: polygon( 50% 0%, 61% 35%, 98% 35%, 68% 57%, 79% 91%, 50% 70%, 21% 91%, 32% 57%, 2% 35%, 39% 35%);
+  transform: matrix(0.97, -0.23, 0.23, 0.97, 0, 0);
+  width: 500px;
+  height: 500px;
+  background: #c4c4c4;
+}
+
+.main--header-description {
+  display: flex;
+  justify-content: space-between;
+  margin-top: 200px;
+}
+
+.main--header-description-p {
+  width: 800px;
+  height: 120px;
+  font-family: lato;
+  font-size: 26px;
+  margin-left: 80px;
+  font-weight: 300;
+}
+
+.main--header-description-request {
+  align-self: flex-end;
+  margin-right: 100px;
+  margin-top: 150px;
+}
+
+.main--header-forYou, .main--header-helpOthers {
+  background: #008326;
+  font-size: 30px;
+  width: 200px;
+  height: 60px;
+  border: none;
+  border-radius: 5px;
+}
+
+.main--header-forYouA, .main--header-helpOthersA {
+  color: white;
+  text-decoration: none;
+}
+
+.main--header-ourVision {
+  background-image: linear-gradient( to left bottom, rgba(1, 1, 1, 0.5), rgba(1, 1, 1, 0.5)), url("https://files.gitter.im/ahmedisam99/pYeg/donation.jpg");
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  padding: 140px 50px 120px 50px;
+}
+
+.main--header-ourVision-h1, .main--header-ourVision-h1 {
+  font-size: 80px;
+  color: white;
+  font-family: Roboto;
+  font-weight: 300;
+  margin-bottom: 50px;
+}
+
+.main--header-ourVision-p, .main--header-ourVision-p {
```

Comment:
@aminalakhsham I think there are a few more opportunities to refactor this css in order to remove some duplication:

If every `p` tag underneath `.main-header` should have some common styles, you can remove their specific classnames and just have a selector like this in your css:
`.main--header p { (all of the common p styles here) }`

If there are a few `p` tags which need extra styling, then you can still do the common styles that I described above, and just add the extra styles where needed:
```
.main--header p {
  (all of the common p styles here)
}
.main--header-how-p {
  //just the color
  color: black;
}
```

It looks like the same could help with the `h1` styles, and maybe some others.

Reply:
thank you for this 
but every one said no tag name in css is this correct ??
@tomduggan85

Reply:
If you're more comfortable with classes, you could do this with a `.main--header-p` class too, so it's just a little more generic and lets you put all of the "common" p styles in one place inside your css file instead of many places.
I would say "no tag names in css" is a good guideline to start with, but doesn't have to be a hard rule (in larger web apps, anyways).  Tag selectors are the slowest selector (slow meaning, how much time does it take for the browser to search the page for those elements when applying your styles), so you have to keep that in mind, but I think only a large website / webapp will actually have a *noticeable* slow-down from that.  Smaller websites, you won't notice.  I've found it helpful to still use tag names in css, just so you don't have to create "artificial" tag selectors like `.my-h1`, it is simpler to just say `h1` in your css.